src/ResourceManager/Storage/ChangeLog.md

```diff
@@ -18,6 +18,13 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Upgrade SRP SDK to 6.3.1
```

Comment:
The version ofthe SRP sdk doesn't mean anything to customers, we should remove this in a later PR.